.prettierrc.json

```
@@ -3,5 +3,6 @@
   "singleQuote": true,
   "trailingComma": "all",
   "printWidth": 100,
-  "proseWrap": "always"
+  "proseWrap": "always",
+  "endOfLine": "lf"
```

it's not common to force `lf` locally. This is what git config is for. I'm trying to keep the same prettier rc with as many defaults as possible across all game-ci repos.

```suggestion
  "proseWrap": "always"
```